### PR TITLE
Rz/1inch UI errors

### PR DESCRIPTION
**Pull request by pacxiu**
# [1inch UI errors/loading](https://app.clubhouse.io/oazo-apps/story/2346/add-error-handling-for-1inch-api)
  
## Changes 👷‍♀️
- add handling 1 inch errors and loading state
- add storybooks
image
image

## How to test 🧪
- run storybooks locally
- visit open/manage multiply flows on Kovan network
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by paszkowskiDamian** [comment]
Is this correct? 
image

On the manage form there was an error message.

**Comment by paszkowskiDamian** [comment]
We also need to figure out whether we have up to date exchange data :) Maybe we need to clear `swap` field on input changes

File: public/locales/en/common.json [hunk 1]
```diff
@@ -248,7 +248,8 @@
       "custom-dai-allowance-amount-less-than-payback-amount": "You need at least an allowance of your payback amount to continue",
       "depositing-all-eth-balance": "You need some ETH to pay for the gas fees. Lower your deposit amount",
       "ledger-enable-contract-data": "Please enable Contract data on the Ledgers Ethereum app Settings",
-      "withdraw-collateral-on-vault-under-debt-floor": "You cannot withdraw from your vault while it is below the minimum required of {{debtFloor}} Dai. To withdraw your collateral, payback your debt fully. Learn more about the Minimum Vault Debt <0>here</0>."
+      "withdraw-collateral-on-vault-under-debt-floor": "You cannot withdraw from your vault while it is below the minimum required of {{debtFloor}} Dai. To withdraw your collateral, payback your debt fully. Learn more about the Minimum Vault Debt <0>here</0>.",
+      "exchange-error": "Could not fetch exchange prices for doing this multiply action."
```
Comment (adamskrodzki) [inline public/locales/en/common.json:252]: I'm not english native but shouldn't it be more like
Could not fetch exchange prices required for this multiply action.

On more general note, who is deciding on texts wisible to the user ?

Comment (pacxiu) [reply public/locales/en/common.json:252]: They were written in task description, so Frank was putting this text.

**Review by paszkowskiDamian — APPROVED** [review]

**Pull request merged**